src/util/DeviceFingerprint.js

```diff
@@ -89,6 +92,11 @@ define(['q', 'okta'], function (Q, Okta) {
       });
       element.append($iframe);
 
+      iFrameTimeout = setTimeout(() => {
+        // If the iFrame does not load, throw an error
+        handleError('service not available');
+      }, 1000);
```

Comment:
I am worried if this is very less. Can we make this higher and disable sign in btn ?

Reply:
I changed it to 2 sec, but I am gathering data to figure out what is the best